packages/vda-xp-reward-client/.env.example

```diff
@@ -0,0 +1,7 @@
+## These are used for test only.
+PRIVATE_KEY = "<Input private key of transaction sender>"
+RPC_URL="<Input RPC url for targeting chain>"
```

Comment:
We can remove this now as it should use default RPC URLs

Reply:
Just not clear.
`PRIVATE_KEY` is the private key of wallet that send the transaction.

And the `RPC_URL` is used in the tes script to manage the mock token.
![image](https://github.com/verida/verida-js/assets/15656252/b8bba7a9-1b6c-4bc7-b67d-7835b6233ac9)

Reply:
If the RPC_URL is only for the. test script, then make that clear. Add a comment in `.env.example` and maybe rename to `RPC_URL_TEST_DEPLOY`

Reply:
There is comment in the `.env.example` file.
![image](https://github.com/user-attachments/assets/36321530-70b5-4c5e-b850-d17d9e26b3b6)